spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java

```diff
@@ -32,6 +41,25 @@ public HandlerMapping webSocketHandlerMapping() {
 
     @Bean
     public WebSocketHandlerAdapter handlerAdapter() {
-        return new WebSocketHandlerAdapter();
+        return new WebSocketHandlerAdapter(webSocketService());
+    }
+
+    @Bean
+    public WebSocketService webSocketService() {
+        TomcatRequestUpgradeStrategy tomcatRequestUpgradeStrategy = new TomcatRequestUpgradeStrategy();
+        tomcatRequestUpgradeStrategy.setMaxSessionIdleTimeout(10000L);
+        tomcatRequestUpgradeStrategy.setAsyncSendTimeout(10000L);
+        return new HandshakeWebSocketService(tomcatRequestUpgradeStrategy);
+    }
+    @Bean
```

Comment:
should be empty line added

Reply:
I do not understand well what is ment with the empty line added here.

Reply:
To have the file formatted similarly. Before each @bean there is an empty line, so there should be also.

Reply:
Thanks @andrzejsydor . I have updated to add the new line.